packages/scene-composer/src/components/three-fiber/DataOverlayComponent/DataOverlayComponent.tsx

```diff
@@ -19,8 +19,7 @@ export const DataOverlayComponent = ({ node, component }: DataOverlayComponentPr
     <group>
       <Html
         className='tm-html-wrapper'
-        // TODO: add position after finding proper way to always display overlay right above tag
```

Comment:
keep this comment here since it's not fixed yet

Reply:
Done